terraform/modules/subdomain/main.tf

```diff
@@ -0,0 +1,47 @@
+variable "zone" {
+  type = "string"
+  default = "govsvc.uk"
+}
+
+variable "name" {
+  type = "string"
+}
+
+provider "aws" { }
+
+provider "aws" {
```

Comment:
Could we add some comments here around why we're using multiple providers? There's a lot of subtle stuff going on in this terraform file and we should document it somewhere/somehow and I think a few inline comments will go a long way to helping that.

Reply:
good plan ... added to follow up ticket comment for kick-off